src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -427,7 +428,21 @@ else if (logger.isDebugEnabled())
     public void describeShallow(ColibriConferenceIQ iq)
     {
         iq.setID(getID());
-        iq.setName(getName());
+        try
+        {
+            if (conferenceName == null)
+            {
+                iq.setName(null);
+            }
+            else
+            {
+                iq.setName(Localpart.from(conferenceName));
```

Comment:
Conference names currently always come from localparts, so in practice this shouldn't contain any forbidden characters (throwing XmppStringprepException), right?

Reply:
Yeah, the idea is that if they need to be put back into a Localpart, they must have come from a Localpart.